cristin-person/src/test/java/no/unit/nva/cristin/person/fetch/FetchCristinPersonHandlerTest.java

```diff
@@ -98,12 +98,11 @@ void setUp() {
     }
 
     @Test
-    void shouldReturnResponseWhenCallingEndpointWithValidIdentifier() throws IOException {
-        var actual = sendQuery(ZERO_QUERY_PARAMS, VALID_PATH_PARAM).getBodyObject(Person.class);
-        var expectedString = stringFromResources(Path.of(NVA_API_GET_PERSON_RESPONSE_JSON));
-        var expected = OBJECT_MAPPER.readValue(expectedString, Person.class);
+    void shouldReturnResponseWhenCallingEndpointWithValidIdentifier() throws Exception {
+        var actual = sendQuery(ZERO_QUERY_PARAMS, VALID_PATH_PARAM).getBody();
+        var expected = stringFromResources(Path.of(NVA_API_GET_PERSON_RESPONSE_JSON));
 
-        assertThat(actual, equalTo(expected));
+        JSONAssert.assertEquals(expected, actual, JSONCompareMode.NON_EXTENSIBLE);
```

Comment:
JSONCompareMode.LENIENT = Accepts more fields than expected to allow for adding new fields without breaking existing test
JSONCompareMode.NON_EXTENSIBLE = 1:1 relation between expected fields and actual fields. All need to match, except the order of fields.